.gitignore

```diff
@@ -0,0 +1,3 @@
+/sites/default/sqlconf.php 
+/interface/main/calendar/modules/PostCalendar/pntemplates/
```

Comment:
I am pretty sure this will break things (is this or above related to composer stuff?)